c2/c2-protocol/c2-protocol-api/src/main/java/org/apache/nifi/c2/protocol/api/AgentInfo.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Apache NiFi - MiNiFi
+ * Copyright 2014-2018 The Apache Software Foundation
```

Comment:
AFAIK, these copyright indicators are not necessary on every file. Only the license header.

Reply:
Ok will try a sweeping removal :)